docker-compose.yml

```diff
@@ -161,11 +166,15 @@ services:
       - '${STABLE_ROOT_PATH:-/home/jus/repos/nextcloud}/stable17:/var/www/html'
       - '${STABLE_ROOT_PATH:-/home/jus/repos/nextcloud}/stable17/apps-extra:/var/www/html/apps-extra'
       - /var/www/html/data
-      - /var/www/html/config
+      - config:/var/www/html/config
```

Comment:
This would share the config folder between the different versions

Reply:
:+1: